src/EasyCaching.HybridCache/HybridCachingProvider.cs

```diff
@@ -671,7 +686,9 @@ public CacheValue<T> Get<T>(string cacheKey, Func<T> dataRetriever, TimeSpan exp
                 _localCache.Set(cacheKey, result.Value, ts);
             }
 
-            TrackCacheStats(cacheKey, result);
+            if (gotValueFromDistributedCache || gotValueFromLocalCache)
+                TrackCacheStats(cacheKey, result);
```

Comment:
```suggestion
            if (gotValueFromDistributedCache || gotValueFromLocalCache)
            {
                OnCacheHit(cacheKey);
            }
            else
            {
                OnCacheMiss(cacheKey);            
            }
```
Странно, что тесты этого не покрывают.